impeller/aiks/canvas.cc

```diff
@@ -156,6 +187,9 @@ void Canvas::Save(bool create_subpass,
             return filter;
           };
       subpass->SetBackdropFilter(backdrop_filter_proc);
+      MipCountVisitor mip_count_visitor;
+      backdrop_filter->Visit(mip_count_visitor);
+      subpass->SetRequiredMipCount(mip_count_visitor.GetRequiredMipCount());
```

Comment:
We can work out the required mip level for the `EntityPass` texture in `Canvas::Save` without look-aheads later on.

- When creating a new subpass:
  - If the new subpass has backdrop filter, assign the _current_ pass (not the child pass being created) a new max mip level: `GetCurrentPass().SetRequiredMipCount(std::max(GetCurrentPass().GetRequiredMipCount(), backdrop_filter_mip_count))`
  - If the `Canvas` paint state has an image filter, then initialize the max mip level of the subpass with the image filter's max mip level: `subpass->SetRequiredMipCount(paint_image_filter_mip_count)`

Reply:
Then, we just use `subpass->required_mip_count_` in the two places where we provision the child pass texture in `EntityPass`.